Makefile

```
@@ -56,3 +56,9 @@ ftest: bin/pytest bin/elastic-ingest
 
 run: install
 	bin/elastic-ingest --debug
+
+docker:
```

```suggestion
docker-build:
```

Maybe call this `docker-build` to keep it consistent with `docker-run`?